tests/test_messages_config/test.pylintrc

```diff
@@ -124,8 +124,6 @@ pycodestyle-ignore =
     E711,  # pylint C0121
     E712,  # pylint C0121
     E722,  # pylint W0702
-    W291,  # pylint C0303
```

Comment:
you shouldn't delete these, they are used to disable redundant checks from `pycodestyle` (it doesn't impact C0303)